src/tiling/plane_region.rs

```diff
@@ -261,6 +261,10 @@ macro_rules! plane_region_common {
           y: sby + tile_sbo.0.y,
         })
       }
+      #[inline(always)]
+      pub fn frame_block_offset(&self) -> PlaneBlockOffset {
+        self.to_frame_block_offset(TileBlockOffset(BlockOffset { x: 0, y: 0}))
```

Comment:
```suggestion
        self.to_frame_block_offset(TileBlockOffset(BlockOffset { x: 0, y: 0 }))
```
Not caught by rustfmt because it's inside a macro probably.